src/auth/ha-authorize.ts

```diff
@@ -170,6 +173,12 @@ class HaAuthorize extends litLocalizeLiteMixin(LitElement) {
     this._authProvider = ev.detail;
   }
 
+  private _computeDirection() {
```

Comment:
Can we add this to the `lit-localize-lite-mixin`?

Reply:
Should be kept in sync with <https://github.com/home-assistant/frontend/blob/dev/src/state/translations-mixin.ts#L189-L202>